package.json

```diff
@@ -5,26 +5,24 @@
   "main": "server.js",
   "scripts": {
     "build-css": "./css-build",
-    "check-coverage": "istanbul check-coverage --config .istanbul.yml",
-    "check-unit-coverage": "istanbul check-coverage --config .istanbul.unit.yml",
-    "coverage-travis": "yarn generate-coverage && yarn check-coverage && yarn upload-coverage",
-    "generate-coverage": "yarn istanbul -- cover _mocha -- --recursive",
-    "generate-unit-coverage": "yarn istanbul -- cover _mocha -- --recursive test/unit",
-    "git-hook": "yarn lint && yarn generate-unit-coverage && yarn check-unit-coverage",
+    "coverage-check": "istanbul check-coverage --config .istanbul.yml",
+    "coverage-check-unit": "istanbul check-coverage --config .istanbul.unit.yml",
+    "coverage-generate": "yarn istanbul -- cover _mocha -- --recursive",
+    "coverage-generate-unit": "yarn istanbul -- cover _mocha -- --recursive test/unit",
+    "coverage-upload": "cat ./coverage/lcov.info | coveralls",
+    "git-hook": "yarn lint && yarn coverage-generate-unit && yarn coverage-check-unit",
     "istanbul": "NODE_ENV=test istanbul",
     "lint": "eslint --ext .js,.json .",
-    "precommit": "yarn git-hook",
-    "prepush": "yarn git-hook && yarn snyk -- test",
+    "lint-watch": "esw --watch .",
     "postrewrite": "yarn git-hook",
-    "snyk-travis": "yarn snyk -- monitor --org=nhsuk",
+    "precommit": "yarn git-hook",
+    "prepush": "yarn git-hook && yarn snyk test",
     "start": "node app.js",
+    "start-watch": "nodemon app.js | ./node_modules/bunyan/bin/bunyan & nodemon -e scss -x yarn build-css -- compact",
     "test": "NODE_ENV=test mocha --recursive test",
+    "test-ci": "yarn lint && yarn coverage-generate",
```

Comment:
Does this command need `&& yarn coverage-check && coverage-upload` adding to it?